.github/workflows/uds.yml

```
@@ -128,4 +152,5 @@ jobs:
       - name: uds run
         run: |
           cd "$UDS_CORE"
+          npm i "$PEPR_TGZ" 
```

```suggestion
          npm install "$PEPR_TGZ" 
```

As a preference, I like to use the long-form of all commands that run in CI for clarity.